laminas/laminas-mail
# Code style fixes in tests

|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Applies code style fixes to tests:
- add return types
- add parameter types
- cleanup dead code
- fixup phpdoc

... see commits for details
